tests/SpecTests/crud/aggregate-merge.json

```diff
@@ -1,7 +1,7 @@
 {
   "runOn": [
     {
-      "minServerVersion": "4.2.0"
+      "minServerVersion": "4.1.11"
```

Comment:
Our spec tests typically do use the exact version when a feature was introduced. This is totally my fault dating back to https://github.com/mongodb/specifications/commit/b10b400be768f6d5df95eddf90fdb710df1975bd#diff-05d2232459eeabeaeec04d301113e3b3R6.

I may as well submit a PR to go back and change this in the main repo.

Reply:
See https://github.com/mongodb/specifications/pull/614, changes there will be brought in with #642 with the temporary commit in this branch being dropped.

Reply:
See https://github.com/mongodb/specifications/pull/614, changes there will be brought in with #642 with the temporary commit in this branch being dropped.

Reply:
Thanks for linking the PR. I forgot to come back and drop a comment here.

Since merged: https://github.com/mongodb/specifications/commit/efe9319a2b8b046d44fcea9661b9e18fdb109031